bin/s2nc.c

```diff
@@ -517,6 +531,21 @@ int main(int argc, char *const *argv)
             GUARD_EXIT(s2n_connection_use_corked_io(conn), "Error setting corked io");
         }
 
+        /* Read in session ticket from previous session */
+        if (ticket_in) {
+            if(get_file_size(ticket_in, &session_state_length) < 0) {
+                fprintf(stderr, "Failed to read file: %s\n", ticket_in);
+                exit(1);
```

Comment:
Kind of what i was referring to in the comment above. It would be nice to be able to use the s2n_safety_macros here since we are defining the functions.

Reply:
I don't think we should import the s2n safety macros. Let me see if I can add one for this case though.